content/enterprise/install/interactive/installer.mdx

```diff
@@ -159,6 +159,8 @@ Terraform Enterprise uses a standard set of ciphersuites by default. If necessar
 
 (This value must be defined in the OpenSSL [cipher list format](https://www.openssl.org/docs/man1.1.1/man1/ciphers.html)
 
+-> Note: The default enabled ciphers for Nginx are `HIGH:!aNULL:!MD5`.
```

Comment:
```suggestion
-> Note: The default enabled ciphers for Terraform Enterprise are `HIGH:!aNULL:!MD5`.
```

We don't mention Nginx anywhere else in this documentation.. I think probably best just to refer to Terraform Enterprise here, rather than exposing that level of detail.